src/frontend/components/form-group/autosave/lib/autosaveBase.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from "component";
+import gadsStorage from "util/gadsStorage";
+
+export default abstract class AutosaveBase extends Component {
+    constructor(element: HTMLElement) {
+        super(element);
+        this.initAutosave();
+    }
+
+    get isClone() {
+        return location.search.includes('from');
```

Comment:
I'd prefer to get this from some other method that doesn't rely on the URL, may data value again